src/components/SectionHeadline.js

```diff
@@ -17,7 +19,13 @@ export const SectionHeadline = props => (
             </span>
         )}
         {props.label}
-        <PageHelper url={getDocsUrl(props.systemVersion, props.sectionKey)} />
+        <AppContext.Consumer>
+            {({ d2 }) => (
+                <PageHelper
+                    url={getDocsUrl(d2.system.version, props.sectionKey)}
```

Comment:
Instead of wrapping this in `AppContext.Consumer`, `getDocsUrl` could just take a single argument: `props.sectionKey`. And `getDocsUrl` can import `getD2` from the `api.js` file. I think this would make the code a bit cleaner.

Come to think of it: it's quite likely that we can use this much simpler pattern throughout the codebase. Perhaps leave this as it is for now and I can work on this in a separate branch.

Reply:
Will start a separate branch for this